packages/grafana-toolkit/bin/grafana-toolkit.dist.js

```diff
@@ -1,5 +1,22 @@
 #!/usr/bin/env node
 
-// This bin is used for cli installed from npm
+const fs = require('fs');
 
-require('../src/cli/index.js').run();
+entrypoint = () => {
+  const defaultEntryPoint = '../src/cli/index.js';
+  // We are running in dev mode. Don't use compiled binaries, rather use the dev entrypoint.
+  if (fs.existsSync(`${process.env['HOME']}/.config/yarn/link/@grafana/toolkit`)) {
+    console.log('Running in linked mode');
+    return `${__dirname}/grafana-toolkit.js`;
+  }
```

Comment:
Not sure this is going to work TBH. This file (grafana-toolkit.dist.js) is used by the published package. And in the published tar there is no grafana-toolkit.js file anymore.

Not sure what problem is this `entrypoint` function trying to solve?

Reply:
There were two requirements.
When developing the github publish, I wanted to be able to:
1) Just run `yarn --link @grafana/toolkit` from the destination and nothing more complicated.
2) If running in linked mode, don't use the compiled version, use the typescript version so I can make changes on the fly and test them on the fly.